REFERENCE.md

```diff
@@ -0,0 +1,1404 @@
+
+Valid values: nth, random
+
+Set the matching mode for statistic matching. Supported modes are `random` and `nth`.
+
+##### `stat_every`
+
+Match one packet every nth packet. Requires `stat_mode => 'nth'`
+
+##### `stat_packet`
+
+Valid values: %r{^\d+$}
+
+Set the initial counter value for the nth mode. Must be between 0 and the value of `stat_every`. Defaults to 0. Requires `stat_mode => 'nth'`
+
+##### `stat_probability`
+
+Set the probability from 0 to 1 for a packet to be randomly matched. It works only with `stat_mode => 'random'`.
+
+##### `mask`
+
+Sets the mask to use when `recent` is enabled.
+
+##### `gateway`
+
+The TEE target will clone a packet and redirect this clone to another
+machine on the local network segment. gateway is the target host's IP.
+
+##### `ipset`
+
+Matches against the specified ipset list.
+Requires ipset kernel module. Will accept a single element or an array.
+The value is the name of the blacklist, followed by a space, and then
+'src' and/or 'dst' separated by a comma.
+For example: 'blacklist src,dst'
+
+##### `checksum_fill`
+
+Valid values: `true`, `false`
+
+Compute and fill missing packet checksums.
+
+##### `mac_source`
+
+Valid values: %r{^([0-9a-f]{2}[:]){5}([0-9a-f]{2})$}i
+
+MAC Source
+
+##### `physdev_in`
+
+Valid values: %r{^[a-zA-Z0-9\-\._\+]+$}
+
+Match if the packet is entering a bridge from the given interface.
+
+##### `physdev_out`
+
+Valid values: %r{^[a-zA-Z0-9\-\._\+]+$}
+
+Match if the packet is leaving a bridge via the given interface.
+
+##### `physdev_is_bridged`
+
+Valid values: `true`, `false`
+
+Match if the packet is transversing a bridge.
+
+##### `physdev_is_in`
+
+Valid values: `true`, `false`
+
+Matches if the packet has entered through a bridge interface.
+
+##### `physdev_is_out`
+
+Valid values: `true`, `false`
+
+Matches if the packet will leave through a bridge interface.
+
+##### `date_start`
+
+Only match during the given time, which must be in ISO 8601 "T" notation.
+The possible time range is 1970-01-01T00:00:00 to 2038-01-19T04:17:07
+
+##### `date_stop`
+
+Only match during the given time, which must be in ISO 8601 "T" notation.
+The possible time range is 1970-01-01T00:00:00 to 2038-01-19T04:17:07
+
+##### `time_start`
+
+Only match during the given daytime. The possible time range is 00:00:00 to 23:59:59.
+Leading zeroes are allowed (e.g. "06:03") and correctly interpreted as base-10.
+
+##### `time_stop`
+
+Only match during the given daytime. The possible time range is 00:00:00 to 23:59:59.
+Leading zeroes are allowed (e.g. "06:03") and correctly interpreted as base-10.
+
+##### `month_days`
+
+Only match on the given days of the month. Possible values are 1 to 31.
+Note that specifying 31 will of course not match on months which do not have a 31st day;
+the same goes for 28- or 29-day February.
+
+##### `week_days`
+
+Valid values: Mon, Tue, Wed, Thu, Fri, Sat, Sun
+
+Only match on the given weekdays. Possible values are Mon, Tue, Wed, Thu, Fri, Sat, Sun.
```

Comment:
The `possible values` section should be removed as it is already stated in `Valid values`.

Reply:
Removed